pipeline-scripts/release.groovy

```diff
@@ -361,7 +361,13 @@ done
 
     if ( ! params.DRY_RUN ) {
         commonlib.shell(script: rsync_cmd)
-        commonlib.shell(script: "ssh -o StrictHostKeyChecking='no' ${MIRROR_HOST} timeout 15m /usr/local/bin/push.pub.sh openshift-v4")
+        timeout(time: 60, unit: 'MINUTES') {
```

Comment:
:thinking: Wouldn't four times retrying with the timeout kept to 15 mins be statistically more likely to yield a positive result?

Reply:
Not sure, but I think there might be a reason the timeout was defined in the process that runs on `use-mirror`. `push.pub.sh` might get reparented to PID1 when jenkins kills the ssh process, and continue to run.